packages/component-library/src/components/plmg-card/plmg-card.tsx

```diff
@@ -79,7 +99,10 @@ export class Card {
     };
 
     return (
-      <div class={'plmg-card'}>
+      <div
+        class={'plmg-card'}
+        style={{ width: this.fullWidth ? 'full-width' : 'fit-content' }}
```

Comment:
width: full-width is not valid css. It's just ignoring this rule.  Could just write `style={{ width: !this.fullWidth && 'fit-content' }}`?

Reply:
Yeah I noticed that actually, was just copying PlmgButton's implementation, but I will adjust here anyway as you are right it is doing nothing lol.